src/includes/getting-started-primer/javascript.svelte.mdx

```diff
@@ -0,0 +1,13 @@
+<Note>
+
+Sentry's Svelte SDK enables automatic reporting of errors and exceptions, as well as performance monitoring for your client-side Svelte apps.
+
+</Note>
+
+<Alert level="info" title="Important">
+
+This SDK is considered **experimental and in alpha state**. It may experience breaking changes. Please reach out on [GitHub](https://github.com/getsentry/sentry-javascript/issues/new/choose) if you have any feedback or concerns. The SDK currently only supports [Svelte](https://svelte.dev/) and is not yet compatible with with [SvelteKit](https://kit.svelte.dev/).
+
+</Alert>
+
+Sentry's Svelte SDK was introduced with version `7.10.0`.
```

Comment:
I also agree that we should just take this out since we don't consistently include it.
```suggestion
```

Reply:
Again one of the things I took from the Remix docs. I think the reason for adding this was because we didn't ship this SDK with v7 originally but only introduced it later. 
Happy to take it out but then I think we should also remove it from the Remix docs.

Reply:
Ah okay, then go ahead and leave it in. That's fine!